Dockerfile

```diff
@@ -28,11 +28,14 @@ COPY setup.py /root/swift_ui/setup.py
 COPY swift_browser_ui /root/swift_ui/swift_browser_ui
 COPY --from=FRONTEND /root/swift_ui/swift_browser_ui_frontend/dist /root/swift_ui/swift_browser_ui_frontend/dist
 
+RUN apk add --no-cache rust cargo \
+    && rm -rf /var/cache/apk/*
+
 RUN pip install --upgrade pip && \
     pip install -r /root/swift_ui/requirements.txt && \
     pip install /root/swift_ui
 
-FROM python:3.8-alpine3.12
+FROM python:3.8-alpine3.13
 
 RUN apk add --no-cache --update bash
 
```

Comment:
Not required for right now, but would make more sense to have just one docker image. If the files differ, they could be added as configmaps instead.

Reply:
for this dockerfile `BACKEND` builds the packages while the layer you pointed has the binaries
If we take into consideration other swift-ui services sure, that could be a way.